FESS-GUI/mainwindow.cpp

```diff
@@ -360,10 +372,46 @@ void MainWindow::on_goButton_clicked()  //when you hit the go button
     ui->textBrowser->append(QString("Flywheel controlled to %1 RPM,"
                                     " %2 rad/sec<sup>2</sup>, %3 rad/sec<sup>3</sup>"
                                     " at %4")
-                            .arg(expectedVelocity).arg(expectedAcceleration).arg(expectedJerk)
+                            .arg(targetVelocity).arg(targetAcceleration).arg(currentExpectedJerk)
                             .arg(QTime::currentTime().toString()));
 }
 
+/*this function manages the slope. It is called every 10ms when you click the go button,
+and stops when you get to your target velocity*/
+void MainWindow::velocitySlope(){
+    if(currentExpectedVelocity <= targetVelocity){ //if the target velocity is greater than the current
+        currentExpectedVelocity += currentExpectedAcceleration/100; //the function runs every 10ms so divide by 100 to get the correct increment
```

Comment:
instead of 100, use 1000 / velocitySlopeTimer->interval(). This is the same thing, but this makes the code more robust to changes. If the timer interval got changed in the future, this function would be wrong, and it might not be clear why. If we use the value from the timer, though, then this function is always right.